content/2_Bronze/Complete_Rec.mdx

```diff
@@ -902,6 +903,7 @@ public class Main {
 	static String g[];
 	static boolean sum[], dif[], c[];
 	static int ans = 0;
+	
 	static void dfs(int r) { //place queen in row r
```

Comment:
Add spacing at the if statement at line 913, and also at the for loop on line 925 please -

Reply:
(Sorry, this is my first time reviewing, so I'm not very good at it)